packages/app-project/pages/404.js

```diff
@@ -0,0 +1,105 @@
+import { ZooHeaderWrapper } from '@components'
+import { ZooFooter } from '@zooniverse/react-components'
+import { Box, Image, Heading, Text, Paragraph, Anchor } from 'grommet'
+import { serverSideTranslations } from 'next-i18next/serverSideTranslations'
+import styled from 'styled-components'
+
+
+const ContainerBox = styled(Box)`
+  position: relative;
+  color: white;
+  z-index: 1;
+`
+
+const Overlay = styled(Box)`
+  position: absolute;
+  top: 0;
+  left: 0;
+  right: 0;
+  opacity: 0.5;
+  z-index: 2;
```

Comment:
We can remove this layer entirely, and replace it with a [CSS filter](https://developer.mozilla.org/en-US/docs/Web/CSS/filter) on the background image. That would make the page structure a lot simpler.